src/main/kotlin/io/github/mojira/arisa/modules/ThumbnailModule.kt

```diff
@@ -61,15 +61,17 @@ class ThumbnailModule(
      * Only consider if:
      * - At start of line or has space or '!' in front
      * - At end of line or has space or '!' behind
-     * - Starting '!' is not followed by space
-     * - Ending '!' is not preceded by space
+     * - Starting '!' is not followed by space or exclamation mark
+     * - Ending '!' is not preceded by space or exclamation mark
```

Comment:
Could you please revert this?
```suggestion
     * - Starting '!' is not followed by space
     * - Ending '!' is not preceded by space
```
These points refer to the negative lookahead `(?!\s)` and negative lookbehind `(?<!\s)` which both have not been changed and actually allow a `!`.

Reply:
Yeah, I forgot to revert these changes.. originally I changed that but it broke stuff. Will do a followup PR.